memote/support/thermodynamics.py

```diff
@@ -0,0 +1,168 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 Novo Nordisk Foundation Center for Biosustainability,
+# Technical University of Denmark.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Supporting functions for checks requiring the eQuilibrator API."""
+
+from __future__ import absolute_import
+
+from sys import version_info
+
+from six import string_types
+
+
+if version_info[:2] >= (3, 5):
+    from equilibrator_api import Reaction, CompoundMatcher
+    COMPOUND_MATCHER = CompoundMatcher()
+
+
+def smallest_compound_ID(kegg_ann_list):
+    """
+    Return an ascending list filtered to contain only KEGG compound IDs.
+
+    KEGG IDs may map to compounds, drugs and glycans prefixed respectively
+    with "C", "D", and "G" followed by at least 5 digits.
+
+    Parameters
+    ----------
+    kegg_ann_list: list
+        A list of mixed KEGG IDs.
+
+    """
+    only_cIDs = [x for x in kegg_ann_list if "C" in x]
+    only_cIDs.sort(key=lambda x: int(x.lstrip('C')))
+    return only_cIDs
+
+
+def get_equilibrator_rxn_string(rxn):
```

Comment:
```suggestion
def get_equilibrator_reaction_string(reaction):
```
My personal preference: Keep external names, i.e., parameters, functions, classes and their methods, and global variables verbose. Use abbreviations inside scope hidden from users.